php/content.md

```diff
@@ -66,6 +65,45 @@ Remember, you must install dependencies for your extensions manually. If an exte
 
 See ["Dockerizing Compiled Software"](https://tianon.xyz/post/2017/12/26/dockerize-compiled-software.html) for a description of the technique Tianon uses for determining the necessary build-time dependencies for any bit of software (which applies directly to compiling PHP extensions).
 
+### Default extensions
+
+Some extensions are compiled by default. As of PHP 7.3 these are:
```

Comment:
Is there a command we could suggest users run instead to get this list for themselves?

Reply:
I got the list with `php -m`

Reply:
I'd rather document the command instead of the list; this list will too easily be out of date.

Reply:
Done.